Bump sheafy

paolobrasolin opened the pull request:
I released `jekyll-sheafy` 0.3.0 which has [some new features](https://github.com/paolobrasolin/jekyll-sheafy/blob/main/CHANGELOG.md#030---2022-02-01) enabling the content of this PR:
* prev/next navigation (I sketched some buttons; they'll need more work for mobile)
* inheriting `macrolib` from the root node (i.e. the lecture, so you can have minibooks with separate macrosets in the same website as we discussed)

paolobrasolin committed:
chore: bump sheafy

paolobrasolin committed:
feat: use inheritable variables to separate macrolib by lecture

paolobrasolin committed:
feat: sketch prev/next navigation

jonsterling commented:
Cool! There's a few issues on my end:

<img width="768" alt="image" src="https://user-images.githubusercontent.com/55057/151945455-5a50a4e1-28e0-4b88-ab37-6c7002705645.png">


1. The wingdings being used cannot be displayed using any font that I have installed on my Mac; so I think it might be good to switch to other symbols. For instance, `»` is available.
2. These titles should be using the `display_title` filter -- since we need to correctly display the title of an untitled result node. (Btw, I wasn't sure if such a filter was the best way to achieve my desired behavior but I just did it because I needed it --- feel free to refactor that functionality; but either way, I would like that logic to be centralized because there are several places in the code where we need to display node titles with numberings.)

paolobrasolin committed:
fix: use display_title for d-pad links

paolobrasolin commented:
Fixed both issues; I also removed the central leaf, it was mostly just annoying to align correctly.
Before:
![image](https://user-images.githubusercontent.com/2499655/151978843-db138026-1f06-4eaf-ad8f-0e8764c8d3ad.png)

After:
![image](https://user-images.githubusercontent.com/2499655/151978617-776ec503-48de-4d1b-a24e-052fb07f5975.png)

The arrows are drawn w/ basic CSS, so everyone will be able to see them.

paolobrasolin committed:
fix: use css not unicode in d-pad

jonsterling reviewed (approved):
This looks great, nice work!
